File: FastCleanApp/app/src/main/java/com/example/fastclean/Models/Utilizador/Funcionario/FuncionarioUpdate.kt
package com.example.fastclean.Models.Utilizador.Funcionario

import com.example.fastclean.Models.Utilizador.Morada
import com.example.fastclean.Models.Utilizador.UtilizadorUpdate
import java.io.Serializable

class FuncionarioUpdate (id: Int,
                         imagem: String,
                         morada: Morada,
                         contacto: Int,
                         ccFile: String,
                         cadastro: String,
                         password: String,
                         cartaDeConducao: String,
                         historicoMedico : String,
                         cvFile : String
): UtilizadorUpdate(id,
    imagem,
    morada,
    contacto,
    ccFile,
    cadastro,
    password
), Serializable {
}

File: FastClean/FastCleanAPI/DTO's/Funcionario/FuncionarioUpdate.cs
using FastCleanAPI.DTO_s.Utilizador;
using FastCleanAPI.Models;
using System.ComponentModel.DataAnnotations;

namespace FastCleanAPI.DTO_s.Funcionario
{
    public class FuncionarioUpdate : UtilizadorUpdate
    {
        
        public IFormFile CartaDeConducao { get; set; }

        
        public IFormFile HistoricoMedico { get; set; }

        
        public IFormFile CvFile  { get; set; }


        public double Preco { get; set; }

        public TipoLimpeza TipoLimpeza { get; set; } 


    }
}
